events/EiffelConfidenceLevelModifiedEvent/simple.json

```diff
@@ -0,0 +1,32 @@
+{
+  "meta": {
+    "type": "EiffelConfidenceLevelModifiedEvent",
+    "version": "1.0",
+    "time": 1234567890,
+    "domainId": "example.domain",
+    "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee"
+  },
+  "data": {
+    "name": "test",
```

Comment:
I think a better example is to use "stabilityTest" or something more descriptive of the CL.